internal/app/user/mongodb_repo.go

```diff
@@ -0,0 +1,120 @@
+package user
+
+import (
+	"context"
+	"dating/internal/app/types"
+	"encoding/base64"
+	"fmt"
+
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	mgo "github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/bcrypt"
+)
+
+var (
+	jwtKey, _   = base64.URLEncoding.DecodeString("dating21")
+	ErrNotFound = errors.New("not found")
+)
+
+type MongoRepository struct {
+	session *mgo.Session
+}
+
+func NewMongoRepository(s *mgo.Session) *MongoRepository {
+	return &MongoRepository{
+		session: s,
+	}
+}
+
+// Sign up method
```

Comment:
Please make a doc  more useful. Ex: This method helps to register new member ...